homeassistant/components/shelly/const.py

```diff
@@ -72,6 +72,18 @@
     "button3": 3,
 }
 
+RGB_EFFECTS = {
+    0: "Off",
+    1: "Meteor shower",
+    2: "Gradual change",
+    3: "Breath",
+    4: "Flash",
+    5: "On/off gradual",
+    6: "Red/green change",
+}
+
+RGB_EFFECTS_INDEX = {"DEFAULT": 6, "SHRGBW2": 3, "SHRGBWW-01": 3}
```

Comment:
This will be very problematic, In my opinion the default should be 3, currently there is only one model which supports more than 3 effect types which is `SHBLB-1`.
I think we should have a basic set of 3 effects, use it for all devices and expand it only for devices which have more than 3 effects and drop the `RGB_EFFECTS_INDEX`

Reply:
We can do the other way around, DEFAULT to 3 and SHBLB-1: 6 ;-)

This coding allow to easily expand/change thing for current or new devices, while maintaining a single list.

Reply:
BTW, you voted on Allterco GitLab for the index solution ;-)

Reply:
DEFAULT changed to 3 as most devices have only 3 effects.